docs/DeveloperGuide.md

```diff
@@ -347,6 +348,37 @@ The unique id of `Person` is stored as a private field `Id` instance in `Person`
 
 Operations with `Id` on `Person` in NetConnect is facilitated through `Model#hasId(Id)` and `Model#getPersonById(Id)`.
 
+### Findnum feature
+
+#### Expected behaviour
+
+The `findnum` command allows users to identify one or more `Person`s from NetConnect using one or more 'Phone' numbers. Exceptional behaviour:
+* If any of the phone numbers provided is invalid, an error message is shown.
+* If there are no `Person`s with the given `Phone`, the display is updated to show an empty list.
+
+<puml src="diagrams/FindNumActivityDiagram.puml" alt="FindNumActivityDiagram" />
+
+#### Current implementation
+Given a command `find 98765432`, the `NetConnectParser` recognises the `find` command and first instantiates a 'FindNumCommandParser' object. It then passes the command string, where each 'number' is validated using the 
+`Phone#isValidPhone(...)` method. Following which, `FindNumCommandParser` creates a predicate called `PhoneContainsNumbersPredicate`, which is then used to create a `Command` object. 
```

Comment:
Shouldn't have line wrap for markdown! Can toggle on soft wrap in Intellij for better readability